### ISSUE TITLE
Error in src/index.js

### ISSUE THREAD

**Issue by 440459012** [monterosalondon/react-native-parallax-scroll#17]
```
if (Animated.ScrollView !== props.scrollableComponent) {
      this.ScrollableComponent = Animated.createAnimatedComponent(
        props.scrollableComponent,
      );
    }
```
in your render func you have <this.ScrollableComponent>.... on return, but it's defined only in a conditional function what is in contructor, it must be at least like this:

```
this.ScrollableComponent = Animated.createAnimatedComponent(
      Animated.ScrollView,
    );
if (Animated.ScrollView !== props.scrollableComponent) {
      this.ScrollableComponent = Animated.createAnimatedComponent(
        props.scrollableComponent,
      );
    }
```
or change default props key from 'scrollableComponent' to 'ScrollableComponent

**Comment by z4o4z** [comment]
Hi, @440459012, thanks for the issue. You right, I forgot about it.
Already fixed in v1.5.3

**Comment by 440459012** [comment]
cool, thanks
